staging/src/k8s.io/apiserver/pkg/endpoints/request/context_shard.go

```diff
@@ -0,0 +1,45 @@
+package request
```

Comment:
do we have the boilerplate for our files?

Reply:
`storage_patch` has `Copyright 2022 The KCP Authors.` Going to add kcp boilerplate

Reply:
done